<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cframe</class>
 <widget class="QMainWindow" name="cframe">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1236</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>cframe</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" rowspan="2">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Bytes:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" rowspan="2">
     <widget class="QSpinBox" name="Sb_Bytes">
      <property name="maximum">
       <number>999</number>
      </property>
     </widget>
    </item>
    <item row="0" column="2" rowspan="2">
     <widget class="QPushButton" name="Btn_Asignar">
      <property name="text">
       <string>Asignar</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QRadioButton" name="RB_PrimerA">
      <property name="text">
       <string>Primer Ajuste</string>
      </property>
     </widget>
    </item>
    <item row="0" column="5" rowspan="2">
     <widget class="QPushButton" name="Btn_Reset">
      <property name="text">
       <string>Resetear Todo</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QRadioButton" name="RB_MejorA">
      <property name="text">
       <string>Mejor Ajuste</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QLabel" name="Lbl_Cola">
      <property name="text">
       <string>Disponibles</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Ocupados</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="3">
     <widget class="QTableWidget" name="TW_Mostrar"/>
    </item>
    <item row="3" column="3" colspan="3">
     <widget class="QTableWidget" name="TW_Disponible"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1236</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
